### PR TITLE
pythonPackages.cmd2: 0.8.0 -> 0.9.1

### PR DESCRIPTION
**Pull request by teto**
###### Motivation for this change
Many new features in cmd2 (subcommands/argparse support...)

The problem is that cmd2 dropped python support so I think we should keep both python2 compatible version (0.8.X) and the newest one. Is there any example of this ? 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
Ipython does the same:

```
ipython = if isPy27 then self.ipython_5 else self.ipython_6;
```

File: pkgs/development/python-modules/wcwidth/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, buildPythonPackage }:
+
+buildPythonPackage rec {
+    name = "wcwidth-${version}";
+    version = "0.1.7";
+
+    src = fetchurl {
+      url = "mirror://pypi/w/wcwidth/${name}.tar.gz";
```
Comment (Mic92) [inline pkgs/development/python-modules/wcwidth/default.nix]: fetchPypi

**Comment by teto** [comment]
done

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -9142,7 +9142,9 @@ in {
    cachetools_1 = callPackage ../development/python-modules/cachetools/1.nix {};
    cachetools = callPackage ../development/python-modules/cachetools {};
 
-  cmd2 = callPackage ../development/python-modules/cmd2 {};
+  cmd2_8 = callPackage ../development/python-modules/cmd2/old.nix {};
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: cmd_2_8
cmd_2_9

**Comment by teto** [comment]
name of the package is cmd2; the versions are respectively 0.8 and 0.9 so I
believe cmd2_8 is better ?

2018-06-14 21:40 GMT+09:00 Frederik Rietdijk <notifications@github.com>:

**Comment by FRidh** [comment]
> name of the package is cmd2; the versions are respectively 0.8 and 0.9 so I believe cmd2_8 is better ? 

Yes I just realized after my comment. It's a bit messy in this situation.

**Comment by teto** [comment]
should I move it to staging considering the number of rebuilds (kina impressed, must be the wcwidth change) ?

**Comment by Mic92** [comment]
No master should be ok.

**Comment by xeji** [comment]
@GrahamcOfBorg  build python36Packages.cmd2_9 python36Packages.cmd2_8 python27Packages.cmd2_8

**Pull request merged**